cscs-checks/libraries/boost/boost_python_check.py

```diff
@@ -2,26 +2,26 @@
 import reframe.utility.sanity as sn
 
 
-@rfm.parameterized_test(['1.65.0', '17.08', '2.7'], ['1.65.0', '17.08', '3.6'])
+@rfm.parameterized_test(['1.67.0', '18.07', '2.7'], ['1.67.0', '18.07', '3.6'])
 class BoostCrayGnuPythonTest(rfm.RegressionTest):
     def __init__(self, boost_version, cray_gnu_version, python_version):
         super().__init__()
         self.descr = ('Test for Boost-%s for CrayGnu-%s with python %s '
                       'support') % (boost_version, cray_gnu_version,
                                     python_version)
         self.valid_systems = ['daint:mc', 'daint:gpu', 'dom:mc', 'dom:gpu']
-        self.valid_prog_environs = ['PrgEnv-cray']
-        python_major_version = python_version.split('.')[0]
+        self.valid_prog_environs = ['PrgEnv-gnu']
```

Comment:
Was it wrong previously that used `PrgEnv-cray`?

Reply:
Yes it was wrong. Nevertheless the boost module loaded PrgEnv-gnu, and now that we have the build system I can see the conflict in the .err file.